ddp-workspace/projects/ddp-singular/playwright-e2e/README.md

```diff
@@ -0,0 +1,99 @@
+# DDP Singular project test automation with Playwright
+
+Playwright end-to-end (E2E) UI integration tests.
+
+
+* See [Playwright Test Runner](https://playwright.dev/docs/api/class-test)
+* See [Playwright API](https://playwright.dev/docs/api/class-playwright)
+
+
+## Set up *playwright-e2e* project
+
+* Playwright [Node.js Requirement](https://playwright.dev/docs/troubleshooting#nodejs-requirements)
+* Install `node.js` and `yarn` if not already installed 
+  * Install the latest version of `Node.js`. See [Node.js Install](https://nodejs.dev/en/learn/how-to-install-nodejs/)
+  * Install the latest version of `yarn`. See [Yarn Install](https://yarnpkg.com/getting-started/install)
+* Clone github ddp-angular project: 
+  * `git clone git@github.com:broadinstitute/ddp-angular.git`
+* Install dependencies:
+  * `cd ddp-angular/ddp-workspace/projects/ddp-singular/playwright-e2e`
+  * `yarn install`
+
+## Test Users
+
+- **DO NOT** store passwords in `.env` files.
+
+- **DO NOT** use these users for local development.
+
+Supple credentials for pre-existing test users (rather than by generating new users) using an environment property file `.env`. See `.env.singular.test` for an
+example.
+
+A `.env` file requires following fields:
+- `baseURL`
+- `sitePassword`
+- `userEmail`
+
+
+## Run Tests Using `Yarn` from the Command-Line
+
+**By default, all tests will run in headless mode.**
+
+**To see the list of available Yarn commands** <div class="text-blue">`yarn run`</div>
+
+### Running Tests on Localhost
+
+Fill out local test user credentials
+
+- Copy `.env.singular.sample`, save as `.env`.
+- Update `.env`
+
+### Examples
+
+* Ask for help <div class="text-blue">`yarn test:e2e --help`</div>
+<div></div>
+
+* Run all tests in parallel in **headless mode** <div class="text-blue">`yarn test:e2e --headless`</div>
```

Comment:
I am not able to run the tests in headless mode.
There is an error:
![image](https://user-images.githubusercontent.com/7396837/191725800-f330b290-5ce0-4c8e-a08b-543d80aa4f86.png)

Reply:
If run the tests without `--headless` option
there is an error about non-existed browser path
`browserType.launch: Executable doesn't exist at  ../AppData\Local\ms-playwright\chromium-1019\chrome-win\chrome.exe`